packages/mui-material/src/Modal/Modal.test.js

```diff
@@ -24,7 +24,7 @@ describe('<Modal />', () => {
 
   describeConformance(
     <Modal open>
-      <div />
+      <div tabIndex={-1} />
```

Comment:
If we are adding this in the tests, I suppose developers also need to add it explicitly in their application code. If yes, that would be a breaking change.

Reply:
Yes it is, and the core of the issue with Modal+FocusTrap.

`Modal` can't just add `tabindex={-1}` to it's child if it's not there because it has no idea where the focus should go.

On the other hand, if we don't add them in the tests `FocusTrap` will complain rightly so because there are no focusable children inside.

So basically this block https://github.com/mui/material-ui/pull/37672/files#diff-2c32b674a4f09cd26c4c5e57ea0ebc2b0a8aad65800b9df4dc391cd7c6654f7eL147-L149 was masking dev errors that `FocusTrap` could've warned about.